src/quo2/components/profile/collectible/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.profile.collectible.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.profile.collectible.style :as style]
+            [react-native.core :as rn]))
+
+(defn plus-n-tile
+  [n]
```

Comment:
I recommend using more descriptive parameter names, `n` is hard to understand at a first glance.

Reply:
imo here it's fine no ? plus-n function and we pass n

Reply:
Yeah I would rename the function name too, but just a detail to make it more understandable, not a big deal

Reply:
ok! renamed to `remaining-tiles` and the parameter is `amount` in 08c4d35e0d452ef73b91482837eb8d67721542d2